inference-engine/src/offline_transformations/src/moc_transformations.cpp

```diff
@@ -45,12 +48,20 @@ bool ngraph::pass::MOCTransformations::run_on_function(std::shared_ptr<ngraph::F
     manager.register_pass<ngraph::pass::ConvertQuantizeDequantize>();
     manager.register_pass<ngraph::pass::SimplifyShapeOfSubGraph>();
 
+    auto transpose_sinking = manager.register_pass<ngraph::pass::GraphRewrite>();
+    transpose_sinking->add_matcher<ngraph::pass::TransposeSinking>();
```

Comment:
Is `TransposeSinking` shape-independent transformation? I mean that we need to make sure that it will not break model reshape-ability.

Reply:
TransposeSinking consists of three passes: TransposeFQReduction, TransposeReduction and TransposeFuse. First two passes depends only on shape rank and the last one depends only on Transpose Constant values. So this passes can't break reshapability. Also we execute all MOC Backend passes on fully dynamic function.